go-controller/pkg/ovn/common.go

```diff
@@ -257,18 +257,32 @@ func (oc *Controller) getNetworkInfoFromOvnAnnotation(ovnAnnotation string, netw
 	return ovnExtraAnnotationMap[networkName]
 }
 
-func (oc *Controller) getNetworkNamesFromPodAnnotations(podAnnotations map[string]string) []string {
-	switchesAnnotation, hasExtraSwitches := podAnnotations["switches"]
+func (oc *Controller) getNetworkNamesFromPodAnnotations(podAnnotations map[string]string, namespace string) []string {
+	networksAnnotation, hasExtraNetworks := podAnnotations["k8s.v1.cni.cncf.io/networks"]
 
-	if hasExtraSwitches {
-		var switches []string
-		err := json.Unmarshal([]byte(switchesAnnotation), &switches)
+	if hasExtraNetworks {
+		var networks []map[string]string
+		err := json.Unmarshal([]byte(networksAnnotation), &networks)
```

Comment:
This is not always the case, de-facto standard allows you to specify networks using a coma delimited string:     

k8s.v1.cni.cncf.io/networks: net-a,net-b,other-ns/net-c

Reply:
We are not going to support this format since we are also going to put the mac/ip information into each network in the following way - https://github.com/kubernetes/community/pull/2665/files#diff-b7df51911eac0f70e97ab917caf1156bR293.